openapi/crm/crm.json

```diff
@@ -0,0 +1,381 @@
+{
+  "openapi": "3.1.0",
+  "x-stoplight": {
+    "id": "iyzxipdhj3szx"
+  },
+  "info": {
+    "title": "CRM",
+    "version": "1.0",
+    "description": ""
+  },
+  "servers": [
+    {
+      "url": "https://prod.apigateway.co/org",
+      "description": "Production"
+    },
+    {
+      "url": "https://demo.apigateway.co/org",
+      "description": "Demo"
+    },
+    {
+      "url": "http://localhost:11001/org",
+      "description": "Local"
+    },
+    {
+      "url": "{local}/org",
+      "description": "Local"
+    }
+  ],
+  "paths": {
+    "/{namespace}/{resourceTypeCode}/meta/fields": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "namespace",
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
+        }
+      ],
+      "get": {
+        "summary": "Get field schema",
+        "tags": [],
+        "responses": {
+          "200": {
+            "description": "This API follows [Open API schema](https://spec.openapis.org/oas/v3.1.0#schema-object) of structure for response.",
+            "content": {
+              "application/json": {
+                "examples": {
+                  "Get crm field": {
+                    "value": {
+                      "fields": [
+                        {
+                          "externalId": "externalId_xxxx",
+                          "type": "FIELD_TYPE_xxxx",
+                          "title": "Last activity date",
+                          "description": "Date and time of last activity"
+                        },
+                        {
+                          "externalId": "externalId_xxxx_xxxx",
+                          "type": "FIELD_TYPE_xxxxx",
+                          "title": "Mobile phone number",
+                          "description": "Mobile phone number of the contact."
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "User Not Found"
+          }
+        },
+        "operationId": "ListFieldSchema",
+        "description": "Retrieve field schema based on namespace."
+      }
+    },
+    "/{namespace}/{resourceTypeCode}/meta/fields/{fieldId}/options": {
```

Comment:
Since we will be using the externalID in the majority of the use case and rarely the internal CRM fieldID, do want to update name for fieldID?

Reply:
As per the discussion with the team, we will keep the name.